rebar.config

```diff
@@ -0,0 +1,52 @@
+{erl_opts,
+ [ debug_info
+ , warn_export_vars
+ , warn_unused_import
+ , warn_missing_spec
+ , warn_untyped_record
+ ]}.
+
+{deps, [{getopt, "1.0.1"}]}.
+
+{escript_incl_apps, [concuerror, getopt]}.
+{escript_main_app, concuerror}.
+{escript_name, concuerror}.
+{escript_emu_args, "%%! +S1 -boot start_clean -noshell -pa . -pa ebin\n"}.
+
+{pre_hooks,
+ [ {compile, "priv/generate_version_hrl 20 > src/concuerror_otp_version.hrl"}
+ ]}.
+
+{post_hooks,
+ [ {escriptize, "cp \"$REBAR_BUILD_DIR/bin/concuerror\" ./bin"}
+ ]}.
+
+{profiles,
+ [ {default,
+    []}
```

Comment:
default profile is already the rest of the file, so you can probably take it out

Reply:
This one I originally added because I wanted to silence a warning I was getting when running `rebar3 as default` without such a line. I refactored my `Makefile` to not do that, so I removed it.